doc/contents.rst

```diff
@@ -26,6 +26,10 @@ and contributors, MUA developers and privacy enthusiasts.
      a first collection of current autocrypt developments (Jan 2017) with
      various mail programs and some development resources.
 
+:doc:`implementations`
+    table with Autocrypt-capable MUAs level 1 implementation status
+    (Nov 2017)
```

Comment:
Having the date here gives an opportunity for it to get out of sync with the date next to the table itself.  I'd drop the date from the contents.